.github/workflows/stale_issues_high.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Close stale issues with high priority'
+on:
+  schedule:
+    - cron: '00 04 * * *'
```

Comment:
Add one more trigger so that we could trigger this action manually:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

```
on:
  schedule:
    - cron: '00 04 * * *'
  workflow_dispatch:
```